Integrations/integration-OTRS.yml

```diff
@@ -0,0 +1,971 @@
+description: Service management suite that comprises ticketing, workflow automation,
+  and notification.
+configuration:
+- display: OTRS Server URL (e.g. https://demisto.managed-otrs.com)
+  name: server
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: OTRS Credentials
+  name: credentials
+  defaultvalue: ""
+  type: 9
+  required: true
+- display: Trust any certificate (unsecure)
+  name: unsecure
+  defaultvalue: "true"
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: "false"
+  type: 8
+  required: false
+- display: Fetch incidents
+  name: isFetch
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Incident type
+  name: incidentType
+  defaultvalue: ""
+  type: 13
+  required: false
+- display: Queues to fetch tickets from  ("Any" fetches from all queues. CSV supported,
+    e.g., Misc,Raw)
+  name: fetch_queue
+  defaultvalue: Any
+  type: 0
+  required: false
+- display: Fetch tickets in priority
+  name: fetch_priority
+  defaultvalue: ""
+  type: 16
+  required: false
+  options:
+  - 1VeryLow
```

Comment:
options in params exists only from 4.1
please test in 4.0,3.6 or add fromVersion:4.1 or make it short text

Reply:
Added `fromversion`